uprotocol_vsomeip/vsomeip_utransport.py

```diff
@@ -0,0 +1,511 @@
+# -------------------------------------------------------------------------
```

Comment:
Update copyright. Here is the correct header

"""
SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation

See the NOTICE file(s) distributed with this work for additional
information regarding copyright ownership.

This program and the accompanying materials are made available under the
terms of the Apache License Version 2.0 which is available at

    http://www.apache.org/licenses/LICENSE-2.0

SPDX-License-Identifier: Apache-2.0
"""

Reply:
Updated